transports/upgrades/asb-7to8.md

```diff
@@ -31,6 +31,14 @@ See also [Azure Service Bus Transport Topologies](/transports/azure-service-bus/
 Controlling entity creation has been simplified. Instead of having to provide a full implementation of `DescriptionFactory` where all settings on the description object had to be provided, a customizer (`DescriptionCustomizer`) has been introduced where only the customized properties can be changed. `DescriptionCustomizer` can be found on the `Queues`, `Topics` and `Subscriptions` API extension points.
 
 
+## Queue and Subscription MaxDeliveryCount
+
+In version 7 the transport was setting `MaxDeliveryCount` to match immediate retries specified by each endpoint to ensure messages are not dead-lettered accidentally.
```

Comment:
I suggest changing to present tense instead of moving from past to present tense in one sentence: 
- In version 7 the transport sets `MaxDeliveryCount` to match immediate retries specified by each endpoint to ensure messages are not dead-lettered accidentally.